QRCoder/QRCodeGenerator.cs

```diff
@@ -150,6 +150,85 @@ public static QRCodeData GenerateQrCode(string plainText, ECCLevel eccLevel, boo
         return GenerateQrCode(completeBitArray, eccLevel, version);
     }
 
+    /// <summary>
+    /// Calculates the Micro QR code data which than can be used in one of the rendering classes to generate a graphical representation.
+    /// </summary>
+    /// <param name="plainText">The payload which shall be encoded in the QR code</param>
+    /// <param name="eccLevel">The level of error correction data</param>
+    /// <param name="requestedVersion">Set fixed Micro QR code target version; must be -1 to -4 representing M1 to M4, or 0 for default.</param>
+    /// <exception cref="QRCoder.Exceptions.DataTooLongException">Thrown when the payload is too big to be encoded in a QR code.</exception>
+    /// <returns>Returns the raw QR code data which can be used for rendering.</returns>
+    public static QRCodeData GenerateMicroQrCode(string plainText, ECCLevel eccLevel = ECCLevel.Default, int requestedVersion = 0)
+    {
+        if (requestedVersion < -4 || requestedVersion > 0)
```

Comment:
```suggestion
        if (requestedVersion is < -4 or > 0)
```
Just a really minor thing, but in the future this allows Roslyn to produce optimized code for the range test with the constant set.